<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLabelLineAnchorWidgetBase</class>
 <widget class="QgsPanelWidget" name="QgsLabelLineAnchorWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Obstacle Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Label Anchoring</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1,0">
      <item row="2" column="1">
       <widget class="QComboBox" name="mClippingComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Clipping</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;b&gt;Controls the position along the line which labels will be placed close to.&lt;/b&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QgsPropertyOverrideButton" name="mLineClippingDDBtn">
        <property name="text">
         <string>…</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="topMargin">
         <number>10</number>
        </property>
        <item row="0" column="2">
         <widget class="QgsPropertyOverrideButton" name="mLinePlacementDDBtn">
          <property name="text">
           <string>…</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QgsDoubleSpinBox" name="mCustomPlacementSpinBox">
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QComboBox" name="mPercentPlacementComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Text Anchor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QComboBox" name="mAnchorTextPointComboBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QgsPropertyOverrideButton" name="mAnchorTextPointDDBtn">
        <property name="text">
         <string>…</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="mAnchorTextPointHintLabel">
        <property name="text">
         <string>Hint</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Placement Behavior</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QComboBox" name="mAnchorTypeComboBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QgsPropertyOverrideButton" name="mAnchorTypeDDBtn">
        <property name="text">
         <string>…</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="mAnchorTypeHintLabel">
        <property name="text">
         <string>Hint</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsPropertyOverrideButton</class>
   <extends>QToolButton</extends>
   <header>qgspropertyoverridebutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsPanelWidget</class>
   <extends>QWidget</extends>
   <header>qgspanelwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mPercentPlacementComboBox</tabstop>
  <tabstop>mLinePlacementDDBtn</tabstop>
  <tabstop>mCustomPlacementSpinBox</tabstop>
  <tabstop>mAnchorTypeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
